tests/test_co2eq_parameters.py

```diff
@@ -138,6 +160,80 @@ def callback2(ratio, _zone):
             )
         self.check_power_origin_ratios(callback1, callback2)
 
+    def check_contribution_object(self, contribution, zone, mode, contribution_name):
```

Comment:
testing for two things mainly:
* datetimes - if the contributions for a mode are given as a list with datetimes, we verify that each element of the list have a valid datetime + that the list is ordered by these datetimes
* values for the contributions - check that all contributions have a key "value" and that the value is between 0 and 1